src/app/pages/details/phone-view/instructor-resources-pane/instructor-resources-pane.scss

```diff
@@ -41,7 +41,7 @@
             @include set-font(h4);
         }
 
-        .resource-box {
+        .Login to unlockx {
             width: 20rem;
```

Comment:
another weird selector 🤣

Reply:
I think this got resurrected by a merge. 😬